src/Kuzzle.js

```diff
@@ -203,6 +181,22 @@ class Kuzzle extends KuzzleEventEmitter {
     return this.protocol.sslConnection;
   }
 
+  get authenticated () {
+    return this.auth.authenticationToken && !this.auth.authenticationToken.expired;
+  }
+
+  get jwt () {
+    if (!this.auth.authenticationToken) {
```

Comment:
(nitpicking) should always be null if the token is not set, so this test is redundant

Reply:
But if I didn't check if `authenticationToken` is set, I will have TypeError when I try to access `encodedJwt`

Reply:
Oh, yes, I missed the fact that you now stores the entire token object and not only the encoded value :+1: